product/roundhouse.databases.sqlite/roundhouse.databases.sqlite.csproj

```diff
@@ -1,156 +1,87 @@
+    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+        <DebugType>pdbonly</DebugType>
+        <Optimize>true</Optimize>
+        <OutputPath>..\bin\</OutputPath>
+        <DefineConstants>TRACE</DefineConstants>
+        <ErrorReport>prompt</ErrorReport>
+        <WarningLevel>4</WarningLevel>
+        <PlatformTarget>AnyCPU</PlatformTarget>
+        <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+        <Prefer32Bit>false</Prefer32Bit>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Build|AnyCPU' ">
+        <OutputPath>..\..\build_output\RoundhousE\</OutputPath>
+        <DefineConstants>TRACE</DefineConstants>
+        <Optimize>true</Optimize>
+        <DebugType>pdbonly</DebugType>
+        <PlatformTarget>AnyCPU</PlatformTarget>
+        <ErrorReport>prompt</ErrorReport>
+        <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
+        <Prefer32Bit>false</Prefer32Bit>
+    </PropertyGroup>
+    <ItemGroup>
+        <Reference Include="System"/>
+        <Reference Include="System.ComponentModel.DataAnnotations"/>
+        <Reference Include="System.configuration"/>
+        <Reference Include="System.Core" />
+        <Reference Include="System.Data"/>
+        <Reference Include="System.Data.DataSetExtensions"/>
+        <Reference Include="System.Xml"/>
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="EntityFramework" Version="6.1.3" />
+        <PackageReference Include="FluentNHibernate" Version="1.3.0.733" />
+        <PackageReference Include="Iesi.Collections" Version="3.3.2.4000" />
+        <PackageReference Include="NHibernate" Version="3.3.2.4000" />
+        <PackageReference Include="System.Data.SQLite" Version="1.0.105.2" />
+        <PackageReference Include="System.Data.SQLite.Core" Version="1.0.105.2" />
+        <PackageReference Include="System.Data.SQLite.EF6" Version="1.0.105.2" />
+        <PackageReference Include="System.Data.SQLite.Linq" Version="1.0.105.2" />
+    </ItemGroup>
+    <ItemGroup>
+        <Compile Include="..\..\SolutionVersion.cs">
+            <Link>Properties\SolutionVersion.cs</Link>
+        </Compile>
+   </ItemGroup>
+    <ItemGroup>
+        <ProjectReference Include="..\roundhouse\roundhouse.csproj" />
+    </ItemGroup>
+   <ItemGroup>
+        <None Include="App.config"/>
+    </ItemGroup>
+    <ItemGroup>
+        <EmbeddedResource Include="$(NuGetPackageRoot)System.Data.SQLite\1.0.84\lib\net45\System.Data.SQLite.dll">
+            <Link>assemblyresource\System.Data.SQLite.dll</Link>
+        </EmbeddedResource>
+    </ItemGroup>
+    <Import Project="$(NuGetPackageRoot)System.Data.SQLite.Core\1.0.105.2\build\net45\System.Data.SQLite.Core.targets"
+            Condition="Exists('$(NuGetPackageRoot)System.Data.SQLite.Core\1.0.105.2\build\net45\System.Data.SQLite.Core.targets')"/>
 </Project>
```

Comment:
Are the SQLite Interops not necessary anymore? They were native DLLs and unable to be merged, so they were embedded and unpacked at runtime.

Reply:
I really don't know, I might have missed something. That is a challeng, as I don't know if we have a working integration test environment for all DBs. Do we? If not, maybe we should set up something. People task well of containers. Maybe we could use them for this? (just an idea - we should create a separate issue for this)

Reply:
We should set something, that would be good. AppVeyor would give us this integration testing for cheap.

Reply:
If we are not sure, best to leave those interop dlls there and let them embed in the binary.

Reply:
I like cheap (at least on my spare time. Daytime I like paying for good tools)